### PR TITLE
Added login/logout route and profile button

### PR DESCRIPTION
**Pull request by MuhammadAbdullah-hash**
## What do these changes do?
User can now login/logout by clicking buttons. Once logged in it user has the option to logout and his/her name is displayed in a button wich we can use to navigate to profile section later on when that's added

### CODE REVIEW

**Review by hariramjp777 — CHANGES_REQUESTED** [review]
Hi @MuhammadAbdullah-hash, Thanks for your contribution. Mention the related issue in your first comment.
```md
Fixes #<issuenumber>
```
Please add screenshots or gif when you're making PR. :+1:

**Review by khushishikhu — COMMENTED** [review]
@MuhammadAbdullah-hash  I was testing this PR and the localhost:8000 isn't working. Please look into this error.

File: Woodog/templates/partials/base.html [hunk 1]
```diff
@@ -141,7 +141,7 @@
         </li>
       </ul>
 
-      <a class="nav-link" id="sign-up" href="{% url "register" %}">Sign up</a>
+      <a class="nav-link" id="sign-up" href = "{% url "woodogdata" %}" >Sign up</a>
```
[the remark this answers is not in the document]
Comment (khushishikhu) [reply Woodog/templates/partials/base.html:144]: @hariramjp777  yes you are right, the line you have suggested would work

**Comment by MuhammadAbdullah-hash** [comment]
Sure @khushishikhu @hariramjp777 will make the changes. THANKS for feedback :)

**Pull request closed without merging**